src/MapfileStyleParser.ts

```diff
@@ -810,6 +810,15 @@ export class MapfileStyleParser implements StyleParser {
       mapfileLayerLabelItem = mapfileClass.text ? mapfileClass.text : mapfileLayerLabelItem;
       mapfileClass.labels.forEach((mapfileLabel) => {
         mapfileLabel.text = mapfileLabel.text ? mapfileLabel.text : mapfileLayerLabelItem;
+
+        // Set Icons in front of the associated label
+        const labelIcons =  this.getIconsFromMapfileLabel(mapfileLabel);
+        if (labelIcons) {
```

Comment:
this can be removed with the change suggested below

Reply:
Removed the `if`